urbanairship-accengage/src/main/java/com/urbanairship/accengage/AccengageMessage.java

```diff
@@ -134,12 +138,12 @@ private AccengageMessage(@NonNull PushMessage message) {
      * @throws IllegalArgumentException if the push message is not an Accengage push.
      */
     @NonNull
-    public static AccengageMessage fromAirshipPushMessage(@NonNull PushMessage message) {
+    public static AccengageMessage fromAirshipPushMessage(@NonNull PushMessage message, @NonNull AirshipConfigOptions configOptions) {
```

Comment:
This is a breaking API change. Can we do without modifying this class and just fallback to the config option values in the AccengageNotificationProvider?